### PR TITLE
pass: add plugins to support importing and updating

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

There are some neat plugins for pass and while the importer requires ruby and python to run, I don't think we should force them in order to not inflate the size of the closure.

This way they are available (and working if ruby/python is in the path).

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by peterhoeg** [comment]
Cc:  @lovek323 @the-kenny

**Comment by fpletz** [comment]
Hmm, I would prefer a more dynamic solution that includes all dependencies for the plugins the user can activate specifically. For instance like the vim or emacs plugins. On the other hand, merging this hack wouldn't probably hurt regular pass users.

**Comment by peterhoeg** [comment]
It is admittedly hackish, but as it's just a few executables I didn't think too much of it. However, I don't want to merge something hackish without someone else's nod of approval.

**Comment by peterhoeg** [comment]
@the-kenny and @lovek323 - as pass maintainers, any comments?

**Review by fpletz — APPROVED** [review]

**Comment by fpletz** [comment]
Sorry, I forgot about this PR. Please go ahead and merge if you want. It's better than before. :)

**Pull request merged**
